source/common/res/features/budget-daily-spending/main.js

```diff
@@ -0,0 +1,127 @@
+(function poll() {
+  if (typeof ynabToolKit !== 'undefined' && ynabToolKit.pageReady === true) {
+    ynabToolKit.dailySpending = (function () {
+      return {
+        budgetView: ynab.YNABSharedLib.getBudgetViewModel_AllBudgetMonthsViewModel()._result,
+
+        invoke() {
+          let categories = ynabToolKit.dailySpending.getCategories();
+          let categoryName = ynabToolKit.dailySpending.getInspectorName();
+          let masterCategoryViewId = $('ul.is-checked').prevAll('ul.is-master-category').attr('id');
+          let masterCategory = ynabToolKit.shared.getEmberView(masterCategoryViewId).get('data');
+          let masterCategoryId = masterCategory.get('categoryId');
+
+          categories.forEach(function (f) {
+            if (f.name === categoryName && f.masterCategoryId === masterCategoryId) {
+              ynabToolKit.dailySpending.updateDailySpending(f);
+
+              return false;
+            }
+          });
+        },
+
+        observe(changedNodes) {
+          if (changedNodes.has('navlink-budget active') ||
+              changedNodes.has('budget-table-cell-available-div user-data') ||
+              changedNodes.has('budget-inspector')) {
+            ynabToolKit.dailySpending.invoke();
+          }
+        },
+
+        addBudgetVersionIdObserver() {
+          let applicationController = ynabToolKit.shared.containerLookup('controller:application');
```

Comment:
hmm, this makes me think you might want to build your feature off of the new `framework` branch...

I've added a listen in the new framework to this value changing.  This is only a suggestion and is not at all required.